docs/content/getting-started/install.mdx

```diff
@@ -11,6 +11,9 @@ Dagster is a a data orchestrator. Learn what Dagster is all about on our [homepa
 
 ## Installing Dagster
 
+- `dagster` is the command line interface (CLI) tool to run Dagster. It also is the stateless, single-node, single-process and multi-process execution engines
```

Comment:
❓ Should this be `execution engine`? The verb in this sentence is singular, so there's a mismatch between it and the predicate

Additionally, this list item doesn't have punctuation but the next one does. We don't have a set style, but it generally doesn't matter as long as it's consistent.